cms/templates/js/mock/mock-settings-page.underscore

```diff
@@ -107,4 +107,31 @@
     </li>
   </ol>
 </section>
+
+<section class="group-settings course-learning-info">
+    <header>
+        <h2 class="title-2">Learning Outcomes</h2>
+        <span class="tip">Add the learning outcomes for this course</span>
+    </header>
+    <ol class="list-input enum">
+      <li class="course-settings-learning-fields"></li>
+      <li>
+          <a href="javascript:void(0)" class="add-course-learning-info">Add another learning outcome</a>
```

Comment:
Remember to update this mock when you change the HTML to get rid of anchor links.